tests/typeSupport_test/TypeSupport_to_include.idl

```diff
@@ -0,0 +1,6 @@
+
+@topic
+struct MySample2 {
+    @key string a_string;
+    double a_double;
+};
```

Comment:
This file and a few others are missing the final newline at the end. Github shows this as a red marker at the end of the file.